src/run-single-fargate-task.py

```diff
@@ -61,6 +61,15 @@ def create_task_definition(
     client = boto3.client("ecs")
     task_family = f"{state.replace(' ', '_') if state else 'one-off-task'}-{date_time_obj.strftime('%Y%m%d%H%M')}"
     shellscript = (
+        "cat <<EOF >> /tmp/workspace/error_header.log\n"
+        "---------------\n"
+        "THE FOLLOWING IS JUST AN EXCERPT - FULL LOG AVAILABLE AT:\n"
+        "\n"
+        f"https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logStream:group=/aws/ecs/{task_name};prefix={task_family}-main;streamFilter=typeLogStreamPrefix\n"
```

Comment:
No guarantee that this is running in eu-west-1

Reply:
That's true, but eu-west-1 is hardcoded into the rest of the file as well.. I can fix that here, or we can do that in another PR